frontend/src/pages/NewChainSignup.tsx

```diff
@@ -232,6 +241,17 @@ const Signup = () => {
                   </Button>
                 </div>
               </Form>
+              <div className={classes.formHelperLink}>
+                <Typography component="p" className="text">
+                  Troubles with the signup? Contact us
```

Comment:
Please add a translation for this piece of text, key e.g. `troublesWithTheSignupContactUs`